### DISCUSSION TITLE
Allow using a windows image

### DISCUSSION THREAD

**Discussion by ArjixWasTaken** [community/community#45382] [Codespaces]
From my understanding codespaces are exclusively using Linux.

That doesn't sound bad until you want to work with Rust...

It is well known that rust-analyzer crashes a lot on linux, and I have been experiencing that a lot when I'm using codespaces.

Yes I know this is smth that the folks maintaining rust-analyzer should fix, but they haven't done so for like 2 years now.

Being able to use a windows image would solve that problem and would also be great for testing a program on other platforms (since a GitHub action is not interactive).

Yes I know this might be a bit of a pain to implement, and I will accept "no" as an answer.

I just felt the need to suggest this capability.

Cheers,
Arjix

**Comment by jge162** [comment]
Nice suggestion @ArjixWasTaken 👍
